packages/react-components/src/components/TogetherModeOverlay.test.tsx

```diff
@@ -0,0 +1,395 @@
+  });
+
+  test('Confirm displayName is rendered when hand is raised', () => {
+    //   const raisedHand: RaisedHand = {
+    //     raisedHandOrderPosition: 1
+    //   };
+    const localParticipant = createLocalParticipant({
+      videoStream: { isAvailable: true, renderElement: createVideoDivElement() }
+    });
+    const remoteParticipants = Array.from({ length: 1 }, (_, index) =>
+      createRemoteParticipant({
+        userId: `remoteParticipant-${index + 1}`,
+        displayName: `Remote Participant ${index + 1}`,
+        videoStream: { isAvailable: true, renderElement: createVideoDivElement() }
+      })
+    );
+
+    const togetherModeSeatPositions: VideoGalleryTogetherModeParticipantPosition = {
+      [localParticipant.userId]: { top: 0, left: 0, width: 0, height: 0 },
+      ...Object.fromEntries(
+        remoteParticipants.map((participant, index) => [
+          participant.userId,
+          { top: index * 10, left: index * 10, width: 100, height: 100 }
+        ])
+      )
+    };
+    const togetherModeOverLayProps = {
+      emojiSize: 16,
+      reactionResources: [] as ReactionResources,
+      localParticipant,
+      remoteParticipants,
+      togetherModeSeatPositions
+    };
+    const { container } = render(<TogetherModeOverlay {...togetherModeOverLayProps} />);
+    const togetherModeSignalContainers = getTogetherModeSignalContainer(container);
+    expect(togetherModeSignalContainers.length).toBe(2);
+  });
+
+  test('Confirm displayName is rendered when participants perform reactions', () => {
+    const localParticipant = createLocalParticipant({
+      videoStream: { isAvailable: true, renderElement: createVideoDivElement() }
+    });
+    const remoteParticipants = Array.from({ length: 1 }, (_, index) =>
+      createRemoteParticipant({
+        userId: `remoteParticipant-${index + 1}`,
+        displayName: `Remote Participant ${index + 1}`,
+        videoStream: { isAvailable: true, renderElement: createVideoDivElement() }
+      })
+    );
+
+    const togetherModeSeatPositions: VideoGalleryTogetherModeParticipantPosition = {
+      [localParticipant.userId]: { top: 0, left: 0, width: 0, height: 0 },
+      ...Object.fromEntries(
+        remoteParticipants.map((participant, index) => [
+          participant.userId,
+          { top: index * 10, left: index * 10, width: 100, height: 100 }
+        ])
+      )
+    };
+    const togetherModeOverLayProps = {
+      emojiSize: 16,
+      reactionResources: [] as ReactionResources,
+      localParticipant,
+      remoteParticipants,
+      togetherModeSeatPositions
+    };
+    const { rerender, container } = render(<TogetherModeOverlay {...togetherModeOverLayProps} />);
+
+    let renderedReactions = getTogetherModeReactions(container);
+    expect(renderedReactions.length).toBe(0);
+
+    const updatedLocalParticipant = {
+      ...localParticipant,
+      reaction: { reactionType: '👍', receivedOn: new Date() }
```

Comment:
To properly represent the reaction type this should be an enum `'like' | 'heart'` and so on